### DISCUSSION TITLE
关于自定义端口-P

### DISCUSSION THREAD

**Discussion by nimendezaige** [ttttupup/wxhelper#266] [General]
尝试了很多都不行哈，都是Exit Code: 1
我是要多开的，怀疑人生了，有没有大哥帮帮忙
作者说用 specify-port. 分支的dll，我怀疑是作者没打包成dll，让我们自己打包，可我一个java搞不定这个....
还有另一点就是作者说不修复×86的问题，是不是我的微信客户端是32的问题导致了，望解答

**Comment by ttttupup** [comment]
命令行式的只有这个版本的specify-port.支持，最新的配置文件修改即可，多看文档

**Reply by nimendezaige** [reply]
你就是我亲爹啊！！！能直白的告诉我specify-port 分支下的哪个文件哪个命令吗
ConsoleApplication.exe -i WeChat.exe -p d:/wxhelper.dll -P 10444
都不行

**Reply by nimendezaige** [reply]
或者告诉我 specify-port 分支你有打包成dll吗，还是需要我自己打包

**Comment by nimendezaige** [comment]
搞定了，https://github.com/ttttupup/wxhelper/releases/tag/3.9.0.28-specify-port-V13
这里有编译好的wxhelper.dll
